docs/docs/07-forms.md

```diff
@@ -35,7 +35,7 @@ Like all DOM events, the `onChange` prop is supported on all native components a
 
 ## Controlled Components
 
-An `<input>` with `value` set is a *controlled* component. In a controlled `<input>`, the value of the rendered element will always reflect the `value` prop. For example:
+An `<input>` with a `value` prop is a *controlled* component:
```

Comment:
Right, I'm fine with the second paragraph changing, but the first such paragraph/sentence that I highlighted has no such ambiguity.

But if it's still a concern with the first paragraph, an alternate wording could be:
> Specifying a `value` prop on an `<input>` makes it a controlled input component.  In a controlled `<input>`, the value of the rendered element will always reflect the `value` prop. For example:

I mostly just want to give the reader some context before we throw some code at them; otherwise they're almost guaranteed to miss what we're trying to show them and have to go back to read it again.  We give them a complete description of what's going on, show them an example, and explain the example.  If they forget what a controlled component is in the future, they can open the page, read the first two sentences and say "oh yeah, I remember now" and close the page without digging through a novel to get to the implication part.